src/control/cmd/daos/container.go

```diff
@@ -0,0 +1,834 @@
+//
+// (C) Copyright 2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
```

Comment:
All "daos cont" operation handlers? It would be great to help identifying what are helper functions vs real operation handler. We have "static" in C and we usually add a comment to say that this is the handler. Would be great to have some equivalent here.